src/adapters/secondary/displayNewlineSentence.ts

```diff
@@ -0,0 +1,16 @@
+import { DisplaySentences } from '../../fizzbuzz/domain/DisplaySentences';
+import { GetSentencesResponse } from '../../fizzbuzz/usecase/getSentences/getSentences.response';
+
+export class DisplayNewlineSentence extends DisplaySentences {
+
+  constructor(private readonly from: number,
+              private readonly count: number) {
+    super();
+  }
+
+  display() {
+    const responses: GetSentencesResponse = this.useCase.execute({ from: this.from, count: this.count });
+    return responses.sentences.join('\n');
```

Comment:
`DisplayInlineSentence ` and `DisplayNewlineSentence` does exactly the same except for the `join` parameter. 
`useCase.execute()` could be invoked in abstract class to avoid duplication

Reply:
display could take `from`/`count` as method params rather than ctor params so you dont have to create a new instance then call its `display` method
alternative is to pass `from`/`count` to `super()` so that each `DisplaySentence` implementation has the same behavior